### PR TITLE
Adding c projects

### LINKED ISSUE #428: adding my C projects to website
Hi! I would like to add my C projects to the website. 
"I'm in GSSOC'23"

### PR DESCRIPTION
**Pull request by akkshayTandon**
## Related Issue

closes #428 

## Description

added my C programs

## Screenshots

nil

### CODE REVIEW

File: src/DB/projects.json [hunk 1]
```diff
@@ -784,20 +784,20 @@
...
     "Projects": [
       {
-        "link": "https://github.com/itsme-shivamkumar/desume",
-        "title": "Desume | Resume Builder For Developers",
-        "description": "Desume Editor is a React-based web app that allows developers to edit their resumes with ease. With Desume Editor, developers can create or modify resumes with HTML and CSS directly in the app.",
-        "tech": ["React", "Firebase", "JavaScript", "CSS", "HTML", "Framer Motion"]
+        "link": "https://github.com/akkshayTandon/C-Programs",
+        "title": "C-Programs",
+        "description": "This repository contains some useful C programs, each categorized, which can help the college students.",
+        "tech": ["C"]
```
Comment (priyankarpal) [inline src/DB/projects.json:800]: you deleted someone's code could you revert these changes?

**Review by priyankarpal — CHANGES_REQUESTED** [review]

File: src/DB/projects.json (hunk 1 above)
Comment (akkshayTandon) [reply src/DB/projects.json]: please re-review

**Comment by priyankarpal** [comment]
there are some errors. please close this PR & open a new fresh one. make sure to update your fork repo

**Comment by akkshayTandon** [comment]
ok
